frontend/test/metabase/dashboard/dashboard.cy.spec.js

```diff
@@ -112,4 +112,59 @@ var iframeUrl = METABASE_SITE_URL + "/embed/dashboard/" + token + "#bordered=tru
     cy.contains("Remove").click();
     cy.contains("Save").click();
   });
+
+  it("should parse duplicate parameter query params", () => {
```

Comment:
I think we should be a little more conservative with adding Cypress tests for things like this. Does it actually test anything the unit test doesn't cover?

Reply:
The five unit tests cover the actual parsing logic. This Cypress test just checks one case to ensure it's actually using the query param in the browser. i.e. that `location.query` is passed into the component correctly. Though I agree that it's a bunch of code for just that.

Reply:
I removed the test